### ISSUE TITLE
Include balance assertions in hledger print output

### ISSUE THREAD

**Issue by ricklupton** [simonmichael/hledger#255]
I'd like to include balance assertions in the output of `hledger print`, so that I can use it to parse CSV output from my bank and keep track of the running balances. Currently print appears to ignore balance assertions on postings.

Is this something which is intentionally missing, or shall I try to add it to the CSV parser and the `print` output?
##

**Comment by simonmichael** [comment]
I'm not quite understanding, can you give an example ?

I see no reason not to make print show the assertions in its output by default, as ledger does.

**Comment by ricklupton** [comment]
I'd like to get from a CSV file like this:
```csv
"2012/3/22","TRANSFER TO SAVINGS","-10.00","30.00"
"2012/3/23","SOMETHING ELSE","5.50","35.50"
```
with rules:
```
account1 assets:bank:checking
fields date, description, amount, balance
```

so that `hledger -f sample.csv --rules-file sample.csv.rules` gives
```
2012/03/22 TRANSFER TO SAVINGS
    expenses:unknown             10.00
    assets:bank:checking        -10.00 = 30.00

2012/03/23 SOMETHING ELSE
    income:unknown               -5.50
    assets:bank:checking          5.50 = 35.50
```

Currently, the "= 30.00" and "= 35.50" are not included.

I think this needs two changes:

1. make print show assertions, like ledger
2. parse a new "balance" field in csv import

**Comment by simonmichael** [comment]
That's going to be a lot of assertions.. but sounds interesting!

**Comment by simonmichael** [comment]
This was fixed in 1.1.

**Comment by simonmichael** [comment]
Well, the original issue has been resolved. Not the second one (automatic assertions for CSV records).
